prj_scanner/scanner.py

```diff
@@ -0,0 +1,53 @@
+from os import walk
+
+from enums.Language import Language, Runtime
+from models.AppInfo import AppInfo
+
+
+def detect_app_info(url):
+    app_info = AppInfo(Language)
+    if 'http' in url:
+        print('Will scan project with provided url: ' + url)
+    #     nothing's here for now
+    else:
+        print('Will scan project with provided path: ' + url)
+        project_content = scan_folder(url)
+
+        extensions = []
+        for item in project_content:
+            if '.' in item:
+                extensions.append(item.split('.')[-1])
+        extensions = set(extensions)
+        print(extensions)
+    app_info.runtime = define_runtime(extensions)
+
+    return app_info
+
+
+def define_runtime(extensions):
+    lang_list = []
+    for lang in Language.values():
```

Comment:
need help here

Reply:
try this: `lang_list = [(lang, ext) for lanf in Language.values() for ext in extensions if ext in Runtime.extention]`